spec/requests/graphql/mutation/namespace/projects/flows/create_mutation_spec.rb

```diff
@@ -131,6 +131,20 @@
       create(:namespace_member_role, role: namespace_role, member: namespace_member)
     end
 
+    context 'when flow name is taken' do
+      before do
+        create(:flow, name: input[:flow][:name], project: project)
+      end
+
+      it 'returns an error' do
+        mutate!
+
+        expect(graphql_data_at(:namespaces_projects_flows_create, :errors)).to be_present
+        expect(graphql_data_at(:namespaces_projects_flows_create, :flow)).to be_nil
+        expect(graphql_data_at(:namespaces_projects_flows_create, :errors).first['errorCode']).to eq('INVALID_FLOW')
```

Comment:
Not tested, but I think this could also be this

```suggestion
        expect(graphql_data_at(:namespaces_projects_flows_create, :errors, :error_code)).to contain_exactly('INVALID_FLOW')
```